exercises/hamming/lib/hamming.dart

```diff
@@ -1,3 +1,5 @@
 class Hamming {
-  // Put your code here
+  int compute(String a, String b) {
+    // Put your code here
+  }
```

Comment:
This can be reverted. The users will know it needs to be implemented from the tests. Best let them decide the details like the types used.